src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -251,7 +258,7 @@ export default class Resources extends Module {
 		if (ACTIONS.CARVE_AND_SPIT.id === abilityId && !this.buffs.darkArtsActive()) {
 			//great why are we here
 			this._noDACarve += 1
-			this.modifyMana(event.ability, 1200)
+			this.modifyMana(event.ability, CARVE_AND_SPIT_NO_DA_MANA_GAIN)
```

Comment:
If using the `MANA_MODIFIERS` array as mentioned above, you can just do `this.modifyMana(event.ability, MANA_MODIFIERS[abilityId].value)` here which is a bit cleaner I think.